.pre-commit-config.yaml

```diff
@@ -35,27 +35,27 @@ repos:
         args: ["--document-private-items"]
         types: [rust]
         pass_filenames: false
+
+  - repo: https://github.com/kamadorueda/alejandra
+    rev: bb7f2ad3f176aa8e9e2944a10061f7989c8fef17  # frozen: 1.3.0
```

Comment:
@bgins why don't we point to the version here?

Reply:
Yep, sounds good 💯. Will update that.

Reply:
We're holding off on updating this for now. Using the version directly fails.